composer.json

```diff
@@ -12,14 +12,14 @@
     }
   ],
   "require": {
-    "php": ">=7.1",
+    "php": ">=7.4",
     "ext-curl": "*",
     "guzzlehttp/guzzle": "~6.3",
-    "symfony/config" : "~4.3||~5.0",
-    "symfony/dependency-injection": "~3.4||~4.3||~5.0",
-    "symfony/event-dispatcher": "~4.3||~5.0",
-    "symfony/http-kernel": "~3.4||~4.3||~5.0",
-    "symfony/yaml" : "~3.4||~4.3||~5.0"
+    "symfony/config" : "~4.3||~5.0||~6.0",
+    "symfony/dependency-injection": "~3.4||~4.3||~5.0||~6.0",
+    "symfony/event-dispatcher": "~4.3||~5.0||~6.0",
+    "symfony/http-kernel": "~3.4||~4.3||~5.0||~6.0",
+    "symfony/yaml" : "~3.4||~4.3||~5.0||~6.0"
   },
   "require-dev": {
     "atoum/atoum": "~3.4||~4.0",
```

Comment:
couldn't we always use `atoum/atoum: 4.0` if we drop support for older PHP versions?

Reply:
As the bundle now require 7.4 as minimum version 👍🏼